src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```

Comment:
The class 'Truck' should have an access modifier defined. Consider using 'public' if you want this class to be accessible everywhere.

Reply:
Fixed